cli/tests/testdata/run/with_package_json/no_deno_json/main.out

```diff
@@ -1,13 +1,4 @@
 [WILDCARD]package.json file found at '[WILDCARD]with_package_json[WILDCARD]package.json'
 [WILDCARD]
 ok
-[Function (anonymous)] Chalk {
-  constructor: [Function (anonymous)],
-  Instance: [class ChalkClass],
-  supportsColor: false,
-  stderr: [Function (anonymous)] Chalk {
-    constructor: [Function (anonymous)],
-    Instance: [class ChalkClass],
-    supportsColor: false
-  }
-}
```

Comment:
The output was:

```
[Function (anonymous)] Chalk ***
  constructor: [Function (anonymous)],
  Instance: [class ChalkClass],
  supportsColor: false,
  stderr: [Function (anonymous)] Chalk ***
    constructor: [Function (anonymous)],
    Instance: [class ChalkClass],
    supportsColor: false
  ***
***
DEBUG RS - rustls::common_state:495 - Sending warning alert CloseNotify
```

So let's just make most of this a wildcard. What's import here is it's testing if it finds a package.json file or not.